### ISSUE TITLE
Less of an idea, more of a request: Option to make charging icon static and not flashing in menu bar

### ISSUE THREAD

**Issue by thebarbican19** [thebarbican19/BatteryBoi#61]
### Discussed in https://github.com/thebarbican19/BatteryBoi/discussions/53

<div type='discussions-op-text'>

<sup>Originally posted by **adamuniverse** October 14, 2023</sup>
Title really, would be nice to have the option to leave the little lightning bolt always on, instead of it flashing like it currently does!</div>

**Comment by a-tokarev** [comment]
Yes, please. The charging symbol which is _constantly_ flashing in the menu bar is too distracting.
